src/idlcxx/tests/streamer_generator.c

```diff
@@ -19,6 +19,23 @@
 
 #include "CUnit/Theory.h"
 
+#define visual_string_cmp 1
+
+#define test_string_diff3(value_tested,value_tested_for) \
+{ \
+  if (strcmp(value_tested,value_tested_for)) { \
+    if (visual_string_cmp) { \
+      FILE* fa = fopen("/tmp/generated.txt","w"); \
+      FILE * fb = fopen("/tmp/tested.txt","w"); \
```

Comment:
This probably shouldn't be here?

Reply:
You are correct, this is a remnant from when I was testing the code.